demystifying_python/demystifying_python.md

```diff
@@ -21,11 +28,30 @@ After completion of this module, learners will be able to:
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+good_first_module: 
+coding_required: 
+coding_language: python
+
+@sets_you_up_for
+- python_basics_variables_functions
+@end
+
+@depends_on_knowledge_available_in
+- bash_command_line_101
+@end
+
+@version_history 
 
-script:  https://kit.fontawesome.com/83b2343bd4.js
+Previous versions: 
+
+- [1.1.0](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/6f4ca2c54c2fc72a33e5319b20a2ac50562b9ea6/demystifying_python/demystifying_python.md): Removed references to Atom, now sunsetted.
+- [1.0.2](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/466799a081d2cb74d155dc0a26951d3492b81f8c/demystifying_python/demystifying_python.md): Initial Version.
+
+
+@end
+
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
 
-script:  https://code.jquery.com/jquery-3.6.0.slim.min.js
 
 @gifPreload
```

Comment:
We should delete this gifPreload macro, too. It's in the main macros file.

Reply:
Good call, done!